### ISSUE TITLE
[combobox] aria-expanded should be on element with [role=combobox] in aria 1.1 mode

### ISSUE THREAD

**Issue by MathieuPuech** [ing-bank/lion#1295]
## Expected behavior

Aria-expanded on the input of the combobox but missing on the div slot input

## Actual Behavior

- Go to https://lion-web.netlify.app/components/inputs/combobox/features/#autocomplete
- Open aXe DevTools to scan the page
- 2 critical issues found

### Ensures ARIA attributes are allowed for an element's role

Element source

```html
<input id="lion-combobox-q60pebywnn" aria-invalid="false" aria-labelledby="label-lion-combobox-q60pebywnn" aria-describedby="help-text-lion-combobox-q60pebywnn feedback-lion-combobox-q60pebywnn" aria-autocomplete="none" aria-expanded="false" aria-controls="overlay-content--qusx617kkc" style="border: none; outline: none; width: 100%; height: 100%; font: inherit; background: inherit; color: inherit; border-radius: inherit; box-sizing: border-box; padding: 0px;">
```

> Fix the following:
> ARIA attribute is not allowed: aria-expanded="false"

### Ensures elements with ARIA roles have all required ARIA attributes

```html
<div slot="input" class="form-control" role="combobox" aria-haspopup="listbox" aria-owns="overlay-content--qusx617kkc">
```

> Fix the following:
> Required ARIA attribute not present: aria-expanded

## Additional context

I found it while doing render test using axe-core through [pwa-helpers axeReport](https://github.com/Polymer/pwa-helpers/blob/master/src/axe-report.ts). Maybe adding accessibility tests with this could be good to catch these errors?

**Comment by MathieuPuech** [comment]
maybe it's a false error from axe-core?

**Comment by tlouisse** [comment]
Good catch. 
I would have expected this error to pup up here as well: https://github.com/ing-bank/lion/blob/master/packages/listbox/test-suites/ListboxMixin.suite.js#L336.

I think it's always put on the invoker (input element) by the OverlayController, but it should be on the element with role=combobox

For aria 1.0, this should be (and is):
```html
<input role=combobox aria-expanded=true/false>
```


For aria 1.1, this should be:
```html
<div role=combobox aria-expanded=true/false> <input> </div
```

But currenlty it is:
```html
<div role=combobox  <input aria-expanded=true/false> </div>
```

**Comment by tlouisse** [comment]
Will be fixed by https://github.com/ing-bank/lion/pull/1320

@MathieuPuech please let us know whether your issue will be solved after this is merged
